### PR TITLE
Update to mongo 3.0

### PR DESCRIPTION
**Pull request by tmihordea**
## Need
 Update code to mongo 3.0

## Solution
 - update in requirements: pymongo==3.0.3
 - replace safe=True with WriteConcern(w=1)
 - there is only one connection class (remove everything related to replicaSet)

### CODE REVIEW

**Comment by mihneadb** [comment]
Code changes LGTM. Waiting for `setup.py` stuff as well.

**Comment by tmihordea** [comment]
@mihneadb @tiriplicamihai please review

**Comment by mihneadb** [comment]
LGTM

File: setup.py [hunk 1]
```diff
@@ -17,14 +17,14 @@ def read(*filenames, **kwargs):
```
Comment (mihneadb) [inline setup.py]: You need to update the line above to point to README.md

**Comment by tiriplicamihai** [comment]
lgtm

**Pull request merged**